src/transformers/utils/quantization_config.py

```diff
@@ -388,11 +391,14 @@ def __init__(
         self.pad_token_id = pad_token_id
         self.disable_exllama = disable_exllama
         self.max_input_length = max_input_length
+        self.use_exllama_v2 = use_exllama_v2
+        # needed for compatibility with optimum gptq config
+        self.disable_exllamav2 = not use_exllama_v2
```

Comment:
Yeah let's just warn users that `disable_exllama` will be deprecated in future versions here ( or inside `post_init`) and re-think about a better API in the future, but for now I think all good!